packages/vue/src/components/atoms/SfButton/SfButton.stories.js

```diff
@@ -5,7 +5,7 @@ import {
   boolean,
   optionsKnob as options,
 } from "@storybook/addon-knobs";
-import { SfButton } from "@storefront-ui/vue";
+import { SfButton, SfLink } from "@storefront-ui/vue";
```

Comment:
I think you don't need to import SfLinks separately :) it's in SfButton already. WDYT?

Reply:
Yes, that's right.